meta/runtime.yml

```diff
@@ -18,6 +18,8 @@ action_groups:
     - ipam_subnet_info
     - ipam_address_block
     - ipam_address_block_info
+    - ipam_fixed_address
```

Comment:
As per the swagger , its `dhcp_fixed_address`  and `dhcp_fixed_address_info` . this change should be made throughout the PR.